modules/loadbalancer/outputs.tf

```
@@ -1,3 +1,8 @@
+output "lb_id" {
```

```suggestion
output "id" {
```

For input variable e.g. `name` , we are not using `lb_name`, only `name` as we know that it defines name for `loadbalancer` module. Can we use the same approach / convention for output as it's known that we are returning outputs for `loadbalancer` module ?

---

Changed!